src/parsers/manifest/dash/indexes/base.ts

```diff
@@ -299,61 +309,100 @@ export default class BaseRepresentationIndex implements IRepresentationIndex {
   }
 
   /**
+   * `BaseRepresentationIndex` should just already all be generated.
+   * Return `true` as a default value here.
    * @returns {boolean}
    */
   areSegmentsChronologicallyGenerated() : true {
     return true;
   }
 
   /**
-   * @param {Array.<Object>} nextSegments
+   * No segment in a `BaseRepresentationIndex` are known initially.
+   * It is only defined generally in an "index segment" that will thus need to
+   * be first loaded and parsed.
+   * Until then, this `BaseRepresentationIndex` is considered as `unititialized`
```

Comment:
uninitialiazed*

Reply:
That word is too complicated to write and read!
I fixed it and pushed-force